src/filesystem/impls/filer/lib/transforms.js

```diff
@@ -11,7 +11,7 @@ define(function (require, exports, module) {
     var ExtensionUtils = require("utils/ExtensionUtils");
     var marked = require("thirdparty/marked/marked.min");
     var StartupState = require("bramble/StartupState");
-    var Path = BracketsFiler.Path;
+    var Path = require("filesystem/impls/filer/FilerUtils").Path;
```

Comment:
Yeah, it was from a failed rewrite attempt to remove `BracketsFiler`.  I'll revert.